src/services/base64.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ *
+ *  Base64 encode / decode
+ *  http://www.webtoolkit.info/
+ *
+ **/
+export const Base64 = {
+  // private property
+  _keyStr: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=',
```

Comment:
I believe (from the comment) this is copied from somewhere and we cannot simply add the license header, please remove it and add license to the source and dist license file

Reply:
Could we depend on the 3rd party lib directly rather than copying?

Reply:
> Could we depend on the 3rd party lib directly rather than copying?

Sure. I think this is lighter than the 3rd party lib. However, It is okay if we depend on the 3rd party lib.

Reply:
> I believe (from the comment) this is copied from somewhere and we cannot simply add the license header, please remove the license header and add its original license to the source and dist license file

Let me work on it  a bit

Reply:
And keep the license as it used to be. This belongs to them.

Reply:
Copying is fine, if you think this is better, but you need to change the LICENSE in the root, and copy the LICENSE in the `/licenses` folder.

Reply:
Got it